### ISSUE TITLE
re-id feature vector dimension parameter

### ISSUE THREAD

**Issue by rose-jinyang** [ifzhang/FairMOT#95]
Hello
how are you?
I saw that this system uses 128 dimension feature vector for re-id in the paper.
But I found that it is set to 512.
![image](https://user-images.githubusercontent.com/47862419/81506606-19dc5100-932a-11ea-9124-b57d8be3aab5.png)
How should I understand this?
Thanks

**Comment by ifzhang** [comment]
We have experiments on the validation set that proves dim 128 get better results. However, the reid dim of the uploaded baseline model all_dla34.pth is still 512. So we set the default value 512. You can change it to 128 and train your own model and it may get better results.

**Comment by rose-jinyang** [comment]
Thanks
